src/Mail/Message.php

```diff
@@ -382,6 +382,7 @@ protected function buildText(string $html): string
 		]);
 		$text = Nette\Utils\Html::htmlToText($html);
 		$text = Strings::replace($text, '#[ \t]+#', ' ');
+		$text = implode("\n", array_map(static fn(string $line): string => trim($line), explode("\n", str_replace(["\r\n", "\r"], "\n", $text))));
```

Comment:
`static fn(string $line): string => trim($line)` is `trim`, or isn't?

Reply:
It has one difference - PHP calls function internally, so weak typing is used instead of strict types
https://chat.stackoverflow.com/transcript/message/40320863#40320863

Reply:
So, should I change it or will we let it go?